docs/scripting-ref.rst

```diff
@@ -609,7 +610,10 @@ not a size of an area screenshot is taken of. For example, if the viewport
 is 1024px wide then ``splash:png{width=100}`` will return a screenshot
 of the whole viewport, but an image will be downscaled to 100px width.
 
-To set the viewport size use :ref:`splash-set-viewport` method.
+To set the viewport size use :ref:`splash-set-viewport-size`,
+:ref:`splash-set-viewport-full` or *render_all* argument.  *render_all=true* is
+equivalent to running ``splash:set_viewport_full()`` just before the rendering
+and restoring the viewport size afterwards.
```

Comment:
I see that `render_all` argument is (implicitly) tested in render_png emulation script, but it seems the viewport restoring behaviour is not tested.